lib/components/BuilderUI.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useContext, useState } from 'react'
 import { ethers } from 'ethers'
+import { parseInt } from 'lodash'
```

Comment:
You can simply use `parseInt()` anywhere in our JS code. But perhaps I'm missing something by not using lodash's version of it?

Reply:
habit..  safely handles discrepancies among older browsers (irrelevant here) as well as iteration safety within lodash iterator functions (also irrelevant here)..  I will remove the import  :)